code/game/objects/structures/xeno.dm

```diff
@@ -1053,12 +1053,10 @@ TUNNEL
 	user.emote("roar")
 	user.add_filter("resin_jelly_outline", 2, outline_filter(1, COLOR_RED))
 	user.fire_resist_modifier -= 20
-	forceMove(user)//keep it here till the timer finishes
-	user.temporarilyRemoveItemFromInventory(src)
-	addtimer(CALLBACK(src, .proc/deactivate_jelly, user), immune_time)
-
-/obj/item/resin_jelly/proc/deactivate_jelly(mob/living/carbon/xenomorph/user)
-	user.remove_filter("resin_jelly_outline")
-	user.fire_resist_modifier += 20
-	to_chat(user, "<span class='xenonotice'>We feel more vulnerable again.</span>")
+	addtimer(CALLBACK(user, /mob/living/carbon/xenomorph/proc/deactivate_jelly), immune_time)
 	qdel(src)
+
+/mob/living/carbon/xenomorph/proc/deactivate_jelly()
```

Comment:
You need to autodoc new procs.

so here you do this

///Remove the jelly effect from the xenomorph
/mob/living/carbon/xenomorph/proc/deactivate_jelly()

And yes, three /